docs/sdks/sdk-ref-python.md

```diff
@@ -236,6 +238,30 @@ If you're using server-side integration (see [Publisher Integration Guide, Serve
         unmapped_identity = unmapped_identities.get("email1@example.com")
         reason = unmapped_identity.get_reason()
    ```
+### Monitor rotated salt buckets
+1. Create an instance of `IdentityMapClient` as an instance variable or reuse the one from [Map DII to raw UID2s](#map-dii-to-raw-uid2s) 
+   ```py
+   client = IdentityMapClient(base_url, api_key, client_secret)
+   ```
+2. Call a function that takes the timestamp string as input and generates an `IdentityBucketsResponse` object. The timestamp string should be in ISO 8601 format `YYYY-MM-DD[*HH[:MM[:SS[.fff[fff]]]][+HH:MM[:SS[.ffffff]]]]`.
```

Comment:
2. Call a function that takes the timestamp string as input and generates an `IdentityBucketsResponse` object. The timestamp string should be in ISO 8601 format: `YYYY-MM-DD[*HH[:MM[:SS[.fff[fff]]]][+HH:MM[:SS[.ffffff]]]]`.
(just added a colon after format)